public/app/core/components/NestedFolderPicker/NestedFolderPicker.tsx

```diff
@@ -250,7 +250,7 @@ export function NestedFolderPicker({
   });
 
   let label = selectedFolder.data?.title;
-  if (value === '') {
+  if (value === '' || label === 'Root') {
```

Comment:
2 questions:
- what's the underlying `value` when the `label` is `Root`?
- where are we originally setting this label to be `Root`? 

feels a little weird to be overwriting the label when we could just set the label to be correct in the first place 🤔

Reply:
To answer your questions: 
- the value is 'general' which actually is the UID of the folder
- 'Root' is coming from the BE

I talked to Josh about it and I'm going to change the label in the BE instead of overwriting it in the FE.